charts/gadget/templates/daemonset.yaml

```diff
@@ -160,12 +160,38 @@ spec:
               {{- toYaml .Values.capabilities | nindent 14 }}
               {{- end }}
           volumeMounts:
-            - mountPath: /host
-              name: host
+            - mountPath: /host/bin
+              name: bin
+              readOnly: true
+            - mountPath: /host/etc/os-release
+              name: etc-os-release
+              readOnly: true
+            # We need to have read/write as we write NRI config files here.
+            - mountPath: /host/etc/nri
+              name: etc-nri
+            # We need to have read/write as we write OCI config files here.
+            - mountPath: /host/etc/containers/oci
+              name: etc-containers-oci
+            # We need to have read/write as we write NRI binaries here.
+            - mountPath: /host/opt/nri
+              name: opt-nri
+            # We need to have read/write as we write OCI binaries here.
+            - mountPath: /host/opt/hooks/oci
+              name: opt-hooks-oci
+            - mountPath: /host/usr
+              name: usr
+              readOnly: true
+            # We need to have read/write as we write OCI config files here.
+            - mountPath: /host//usr/share/containers/oci/
+              name: usr-share-containers-oci
+            - mountPath: /host/run
+              name: run
+              readOnly: true
```

Comment:
The `run` volume is mounted twice:
- /host/run (RO)
- /run (RW)

Is it on purpose? (It is fine, just asking.)

The mount in /run needs to be mounted in RW mode somewhere to be able to connect to sockets such as:
- /run/docker.sock
- /run/containerd/containerd.sock
- /run/systemd/private

Reply:
I think this discussion is somehow related to https://github.com/inspektor-gadget/inspektor-gadget/issues/450.